src/util/getCursorSortDirection.ts

```diff
@@ -0,0 +1,38 @@
+import State from '../@types/State'
+import ThoughtId from '../@types/ThoughtId'
+import { HOME_PATH } from '../constants'
+import getSortPreference from '../selectors/getSortPreference'
+import head from '../util/head'
+
+/** Retrieves the parent ID from the state's cursor. */
+const getParentIdFromCursor = (state: State): ThoughtId | null => {
+  const cursor = state.cursor
+
+  if (!cursor || cursor.length < 2) {
+    // No cursor or cursor has less than 2 items (no parent)
+    return head(HOME_PATH)
```

Comment:
Can this be simplified? I don't think this is worth holding up the PR, but a minor suggestion:

```suggestion
    return HOME_TOKEN
```